rmf_dashboard_resources/office/main.json

```diff
@@ -1 +1 @@
-{"dispensers": {"coke_dispenser": {"icons": {"coke_dispenser": ""}, "location": {"x": 16.85, "y": -4.8, "yaw": 0, "level_name": "L1"}}, "coke_ingestor": {"icons": {"coke_ingestor": "/dispensers/coke_ingestor/humanIcon.png"}, "location": {"x": 20.95, "y": -6.95, "yaw": 0, "level_name": "L1"}}}, "robots": {"tinyRobot": {"icons": {"tinyRobot": "/robots/tinyRobot/tinyRobot.png"}, "places": {"supplies": [], "tinyRobot2_charger": [], "coe": [], "tinyRobot1_charger": [], "hardware_2": ["coke_ingestor"], "cubicle_2": [], "pantry": ["coke_dispenser"], "station_1": [], "lounge": [], "cubicle_1": [], "hardware_1": [], "station_2": []}}, "deliveryRobot": {"icons": {"deliveryRobot": "/robots/deliveryRobot/deliveryRobot.png"}, "places": {}}}}
\ No newline at end of file
+{"dispensers": {"coke_ingestor": {"icons": {"coke_ingestor": "/dispensers/coke_ingestor/humanIcon.png"}, "location": {"x": 20.95, "y": -6.95, "yaw": 0, "level_name": "L1"}}, "coke_dispenser": {"icons": {"coke_dispenser": ""}, "location": {"x": 16.85, "y": -4.8, "yaw": 0, "level_name": "L1"}}}, "robots": {"deliveryRobot": {"icons": {"deliveryRobot": "/robots/deliveryRobot/deliveryRobot.png"}, "places": {}}, "tinyRobot": {"icons": {"tinyRobot": "/robots/tinyRobot/tinyRobot.png"}, "places": {"supplies": [], "tinyRobot2_charger": [], "coe": [], "tinyRobot1_charger": [], "hardware_2": ["coke_ingestor"], "cubicle_2": [], "pantry": ["coke_dispenser"], "station_1": [], "lounge": [], "cubicle_1": [], "hardware_1": [], "station_2": []}}}}
```

Comment:
Is this change necessary or can it be reverted?

Reply:
There should be no change in this file, as the changes are due to the old remnants.